### ISSUE TITLE
Allow players to set names

### ISSUE THREAD

**Issue by adriantache** [adriantache/Bowling-Scoring#5]

**Comment by Safirah** [comment]
Hey there! Can I start working on this?

**Comment by adriantache** [comment]
Absolutely, have fun with it :) Please submit a pull request when you're done.

**Comment by Safirah** [comment]
Any place in particular that I should put the editText?
I was thinking of changing the TextView with the "Player 1" and "Player 2" to EditViews.
Or maybe create a menu where the players can input their names?

**Comment by adriantache** [comment]
Not sure, depends on what would make sense and also look good. I think changing the TextViews to EditText is a good idea, and you could add a drawableRight to show it's editable. Try it out, see if you like the look.

**Comment by adriantache** [comment]
Or perhaps it would be nicer to have a Button or ImageView next to the TextView that hides TextView, shows EditText, then changes to a submit thing and then hides EditText and updates and then shows the TextView. Seems more elegant than showing a menu or just keeping it EditText.

**Comment by Safirah** [comment]
Thank you for your suggestion, it seems to work quite fine!
But perhaps because my phone screen is quite small it looks a bit odd.

**Comment by Safirah** [comment]
I'll try solving this problem by wrapping everything up on a Scroll View. What do you think?

**Comment by adriantache** [comment]
Could you please post a screenshot?

**Comment by Safirah** [comment]
Sorry, it was actually a mistake on my side. I'll be making the pull request in a couple of minutes :)

**Comment by deezy136** [comment]
Is this portion of the project complete? if not can I work on this?

**Comment by gamino1996** [comment]
Hello, Is this part complete? Let me know as I would like to work on it if not.

**Comment by adriantache** [comment]
Hi there. This part is technically complete, but I just need Safirah to clean up her pull request and resubmit it so I can merge it.

**Comment by KostasThan** [comment]
Hello.. May i download and take a look at the code? first-timer here..

**Comment by adriantache** [comment]
Of course, you can clone the repo without any permissions. And as long as you don't sell it or pass it off as your own there's no need to ask.

**Comment by harshitrjpt** [comment]
any link or doc to set this up and run ? I am a beginner and wanted to contribute or atleast have a lookat it. :)
